src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -465,6 +473,23 @@ public function testDecodeEmptyXml()
         $this->encoder->decode(' ', 'xml');
     }
 
+    protected function getXmlPISource()
```

Comment:
@greg0ire same response. just want to be conform to the existing code. But in fact, you're right ^^